ui/src/app/core-ui/static-properties/static-runtime-resolvable-tree-input/static-tree-input.component.html

```diff
@@ -17,74 +17,214 @@
+                                >folder</i
+                            >
+                            <i
+                                class="material-icons color-primary pr-5"
+                                *ngIf="treeControl.isExpanded(node)"
+                                >folder_open</i
+                            >
+                            <i
+                                class="material-icons pr-5 icon-button"
+                                *ngIf="
+                                    treeControl.isExpanded(node) &&
+                                    hasDataChildren(node)
+                                "
+                                matTooltip="Add all direct children"
+                                >add_circle</i
+                            >
+                            <i
+                                class="material-icons pr-5 icon-button"
+                                *ngIf="selectedNodeId !== node.internalNodeName"
+                                (click)="
+                                    selectedNodeMetadata = node.nodeMetadata;
+                                    selectedNodeId = node.internalNodeName
+                                "
+                                matTooltip="Show details"
+                                >visibility</i
+                            >
+                            <i
+                                class="material-icons pr-5 icon-button"
+                                *ngIf="selectedNodeId === node.internalNodeName"
+                                (click)="
+                                    selectedNodeMetadata = undefined;
+                                    selectedNodeId = undefined
+                                "
+                                matTooltip="Show details"
+                                >visibility_off</i
+                            >
+                            <span class="node-name pr-5"
+                                >&nbsp;{{ node.nodeName }}</span
+                            >
+                        </div>
+                    </div>
+                    <div
+                        [class.sp-tree-invisible]="
+                            !treeControl.isExpanded(node)
+                        "
+                        role="group"
+                    >
+                        <ng-container matTreeNodeOutlet></ng-container>
+                    </div>
+                </mat-nested-tree-node>
+            </mat-tree>
+        </div>
+        <div fxFlex="40" class="tree-input-section" fxLayout="column">
+            <div class="tree-input-header" fxLayoutAlign="start center">
+                <b>Node Details</b>
             </div>
             <div
-                [class.sp-tree-invisible]="!treeControl.isExpanded(node)"
-                role="group"
+                class="node-metadata"
+                *ngFor="let metadata of selectedNodeMetadata | keyvalue"
```

Comment:
Do I get it right here that this is complete agnostic from the metdata that is provided?
So as long as they are provided as a `key-value`  map they are rendered as a tabel?